lstchain/reco/dl0_to_dl1.py

```diff
@@ -244,8 +285,10 @@ def r0_to_dl1(input_filename=get_dataset_path('gamma_test_large.simtel.gz'), out
                     # Some custom def
                     dl1_container.wl = dl1_container.width / dl1_container.length
                     # Log10(Energy) in GeV
-                    dl1_container.mc_energy = event.mc.energy.value
-                    dl1_container.log_mc_energy = np.log10(event.mc.energy.value * 1e3)
+                    if is_simu:
+                        dl1_container.mc_energy = event.mc.energy.value
+                        dl1_container.log_mc_energy = np.log10(event.mc.energy.value * 1e3)
+
                     dl1_container.log_intensity = np.log10(dl1_container.intensity)
                     dl1_container.gps_time = event.trig.gps_time.value
```

Comment:
For the pointing interpolation we need the event time. Since time information is not being written right now to the event.trig.gps_time field, from the time being I've been getting the event time as: 

```
from astropy.time import Time 
start_ntp = event.lst.tel[tel_id].svc.date
ntp_time = start_ntp + event.r0.tel[telescope_id].trigger_time
utc_time = Time(datetime.utcfromtimestamp(ntp_time))
gps_time = utc_time.gps
dl1_container.gps_time = gps_time
```
This should be modified back to the line you wrote when gps_time field is filled up.

Reply:
Ok thanks.
I can integrate this change for real data.
Or you can PR into `vuillaut:real_data` so we aggregate all the changes we need to make for real data analysis and add them at once to lstchain.

Reply:
Ok, I did not know I could do that. I will work on your branch then.